image-layout.md

```diff
@@ -51,9 +51,9 @@ Each object in the `refs` subdirectory MUST be of type `application/vnd.oci.desc
 In general the `mediatype` of this [descriptor][descriptors] object will be either `application/vnd.oci.image.manifest.list.v1+json` or `application/vnd.oci.image.manifest.v1+json` although future versions of the spec MAY use a different mediatype.
 
 **Implementor's Note:**
-A common use case of refs is representing "tags" for a container image.
+A common use case of refs is representing "tags" for a container image, with using "tag" as ref file names.
 For example, an image may have a tag for different versions or builds of the software.
-In the wild you often see "tags" like "v1.0.0-vendor.0", "2.0.0-debug", etc.
+In the wild you often see "tags" like "v1.0.0-vendor.0", "2.0.0-debug", etc which is(not MUST) identical to ref file names.
```

Comment:
This still seems a bit awkward to me.  This [this suggestion][1] is too brief, how about leaving the current there lines as they stand in master, to convey (by sentence):

1. Refs are often used for tags.
2. Images are often tagged by version or build.
3. Tags look like {examples}.

Then add an additional sentence to cover your additional point:

> Those tags will often be represented in an image-layout repository with matching refs like "v1.0.0-vendor.0”, “2.0.0-debug”, etc.

[1]: https://github.com/opencontainers/image-spec/pull/327#issuecomment-248403718